.travis.yml

```diff
@@ -21,12 +36,14 @@ matrix:
 
     - env: IMAGE=dist-x86_64-linux DEPLOY=1
       if: branch = try OR branch = auto
+      after_success: *DEPLOY
```

Comment:
🤔 Could we put that `&DEPLOY` reference into the global `after_success` section (which is configured to do nothing if `$DEPLOY` and `$DEPLOY_ALT` are undefined), so we don't need to add these dozen lines inside the `matrix:`

Reply:
Sure thing, sounds reasonable to me!